bftclient/test/bft_client_test.cpp

```diff
@@ -36,7 +36,7 @@ TEST(msg_receiver_tests, unmatched_replies_returned_no_rsi) {
 
   // Handle the message
   auto source = 1;
-  receiver.onNewMessage(source, reply.data(), reply.size());
+  receiver.onNewMessage(source, reply.data(), reply.size(), source);
```

Comment:
No need to pass the same value twice - let the default value take place. Everywhere in this file.

Reply:
done